dev-packages/browser-integration-tests/suites/public-api/setExtras/consecutive_calls/test.ts

```diff
@@ -10,5 +10,5 @@ sentryTest('should set extras from multiple consecutive calls', async ({ getLoca
   const eventData = await getFirstSentryEnvelopeRequest<Event>(page, url);
 
   expect(eventData.message).toBe('consecutive_calls');
-  expect(eventData.extra).toMatchObject({ extra: [], Infinity: 2, null: null, obj: { foo: ['bar', 'baz', 1] } });
```

Comment:
The `value` under `null` `key` is Infinity

https://github.com/getsentry/sentry-javascript/blob/f664a5f43cb372800d1a3e96c0544d8b8d89dae9/dev-packages/browser-integration-tests/suites/public-api/setExtras/consecutive_calls/subject.js#L8